File: ServerSide/SAMS.REST.API/DataModels/StudentAttendanceEditDataModel.cs
using SAMS.BusinessLogic.Models.Common;
using SAMS.BusinessLogic.Models.StudentAttendances;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace SAMS.REST.API.DataModels
{
    public class StudentAttendanceEditDataModel
    {
        public StudentAttendanceModel StudentAttendance { get; set; }
        public IEnumerable<SelectModel> Subjects { get; set; }
    }
}

File: ClientSide/src/app/pages/studentAttendances/models/studentAttendanceEditDataModel.ts
import { StudentAttendanceModel } from './studentAttendanceModel';
import { SelectModel } from 'src/app/common/models/selectModel';

export class StudentAttendanceEditDataModel {

    public studentAttendance: StudentAttendanceModel;
    public subjects: Array<SelectModel>;

    constructor(
        studentAttendance: StudentAttendanceModel,
        subjects: Array<SelectModel>
    ) {
        this.studentAttendance = studentAttendance;
        this.subjects = subjects;
    }
}
